src/settings.ts

```diff
@@ -116,7 +116,7 @@ function getWeb3ConnectProviderOptions(network: string) {
           package: Torus,
           options: {
             networkParams: {
-              host: "https://xdai.poanetwork.dev",
+              host: "wss://xdai.poanetwork.dev/wss",
```

Comment:
for torus it should stay https

Reply:
Strange, I wonder how the other providers are able to create websocket connections with https?